### ISSUE TITLE
Error in nextTick: "TypeError: Cannot set property 'zIndex' of undefined"

### ISSUE THREAD

**Issue by merry-EIP** [shhdgit/vue-easy-slider#21]
if I hide slider with using v-if or v-else, the error occurs:

> [Vue warn]: Error in nextTick: "TypeError: Cannot set property 'zIndex' of undefined"
> TypeError: Cannot set property 'zIndex' of undefined
>     at VueComponent.handleSetTopItem (index.js?5150:1)
>     at VueComponent.boundFn [as handleSetTopItem] (vue.esm.js?0b33:164)
>     at VueComponent.eval (index.js?5150:1)
>     at Array.eval (vue.esm.js?0b33:655)
>     at nextTickHandler (vue.esm.js?0b33:602)
>     at <anonymous>

my codes are like these:
```
<slider v-if="sites.length<=0">
	<slider-item v-for="(slide, index) in guide.slides" :key="index">
		<div>
			<h3 class="font_violet">{{slide.title}}</h3>
			<div class="wrapper">
				<p class="icon"><img :src="slide.iconSrc"></p>
				<p class="link"><a :href="slide.url">quick menu</a></p>
			</div>
		</div>
	</slider-item>
</slider>
```

**Comment by shhdgit** [comment]
Component is update to 3.1.2
And you can give me a live demo if this bug not be solved

**Comment by merry-EIP** [comment]
I confirmed the issue has resolved in v3.1.2. I appreciate your help!

**Comment by merry-EIP** [comment]
Oops, sorry. The error occurs again. Let me give you a live demo soon.
